tests/plugins/test_kubejobs.py

```diff
@@ -29,11 +30,75 @@ def test_visualize_grafana():
             Returns:
                 None
     """
-    job_id = create_job(MANAGER_URL)
+    job_id = create_job(MANAGER_URL,1)
     grafana_url = wait_for_grafana_url_generation(VISUALIZER_URL, job_id)
     assert requests.get(grafana_url).ok
     stop_job(manager_url=MANAGER_URL, job_id=job_id)
 
+def test_scalling_up():
```

Comment:
Test names should be more specific. What exactly are you testing? For what I understand from your test, I would improve the naming with something like "test_xpto_controller_scales_up". Also, which controller (xpto) are you using to scale up? This is important as well.